bundle/src/main/java/com/adobe/acs/commons/images/transformers/impl/ResizeImageTransformerImpl.java

```diff
@@ -53,14 +53,14 @@
 
     static final String TYPE = "resize";
 
-    private static final String KEY_WIDTH = "width";
-    private static final String KEY_WIDTH_ALIAS = "w";
+    static final String KEY_WIDTH = "width";
+    static final String KEY_WIDTH_ALIAS = "w";
 
-    private static final String KEY_HEIGHT = "height";
-    private static final String KEY_HEIGHT_ALIAS = "h";
+    static final String KEY_HEIGHT = "height";
+    static final String KEY_HEIGHT_ALIAS = "h";
 
 
-    private static final int DEFAULT_MAX_DIMENSION = 50000;
+    public static final int DEFAULT_MAX_DIMENSION = 50000;
```

Comment:
no reason, moving back to private; others are package level since theyre used in ScaledImageTransformerImpl to populate the params map